upgrade/migrate.go

```diff
@@ -79,3 +90,43 @@ func migrateYAMLToJSON() error {
 
 	return nil
 }
+
+func migrateNoNameToName() error {
```

Comment:
The bug was caused by `if biggest == 0` in _utils.GenerateNoteName_, it was my attempt at checking if biggest is empty or not, since nil cannot be used as int. Fixed!